src/index.html

```diff
@@ -1,13 +1,45 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=""
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="imac"
+        class="photo product-card__photo"
+      >
+      <div class="model-name product-card__model-name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
+      <div class="model-code product-card__model-code">Код товара: 195434</div>
+      <div class="reviews product-card__reviews">
+        <div class="stars reviews__stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="text-reviews reviews__text-reviews">Отзывов: 5</div>
+      </div>
+      <div class="price product-card__price">
+        <div class="price-text price__price-text">Цена:</div>
+        <div class="amount price__amount">69 999 грн</div>
+      </div>
+
+      <a class="button product-card__button" data-qa="hover">
+        <div class="button__text">КУПИТЬ</div>
+      </a>
```

Comment:
You should never put block elements inside inline elements. It may break your page layout with certain styles.
I don't think that separate tag for text is needed here. Just write it inside `a`